Automattic/woocommerce-payments
# Checkout page style tweaks

Fixes platform issue 640

#### Changes proposed in this Pull Request
Address all the following style tweaks:

- [x] "Remember my details" headline should match the h3 styles from the rest of the checkout
- [x] Form input styles should be inherited from the theme (checkbox and mobile phone number entry field)
- [x] Body copy in the expanded section should be 0.875 times the size of the body copy
- [x] Body copy and icons should inherit the body copy colors from the checkout
- [x] Ensure vertical spacing between elements is consistent and of reasonable proportions across a variety of themes

The credit for this work is from @brettshumaker. I take his `.diff` file and `git apply` to latest `develop`.

#### Testing instructions
1. Use this branch. Checkout an item.
2. Go to http://localhost:8082/checkout/ and make sure all the bullets above are addressed.
3. Click "Use a new payment method" 
4. Click "Remember your details", enter phone number
5. Proceed to checkout.

-------------------


**Post merge**
